$NetBSD: patch-Makefile,v 1.1 2012/11/17 11:50:14 ryoon Exp $

- Install Coccilib man page just as Coccilib.3 (no need to have a custom
  `cocci' suffix)

--- Makefile.orig	2019-09-25 09:19:54.000000000 +0000
+++ Makefile
@@ -318,7 +314,7 @@ install-man :
 	$(INSTALL_DATA) docs/spatch.1 $(DESTDIR)$(MANDIR)/man1/
 	$(INSTALL_DATA) docs/pycocci.1 $(DESTDIR)$(MANDIR)/man1/
 	$(INSTALL_DATA) docs/spgen.1 $(DESTDIR)$(MANDIR)/man1/
-	$(INSTALL_DATA) docs/Coccilib.3cocci $(DESTDIR)$(MANDIR)/man3/
+	$(INSTALL_DATA) docs/Coccilib.3cocci $(DESTDIR)$(MANDIR)/man3/Coccilib.3
 
 .PHONY : install-spgen
 install-spgen : tools/spgen/source/spgen$(TOOLS_SUFFIX)
